app/src/main/java/com/example/shareyourtrip/PostAdapter.java

```diff
@@ -25,13 +29,19 @@
 
         //This will be a list of user posts to display to the users
         private List<Post> postList;
+        private List<Post> favPostList;
```

Comment:
Why add another list; the postList is passed into the postAdapter through it's constructor in the activity it's in. The users list of favorite posts should be pulled from the database and passed into the postAdapter's constructor in the **FavoriteActivity**.